recipes/rapidgzip/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "rapidgzip" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rapidgzip-{{ version }}.tar.gz
+  sha256: a166b86e7ba4e6091c8c4fdd390963d262ad90ec37a5843e84e115b28381893d
+
+build:
+  skip: true  # [py2k]
+  entry_points:
+    - rapidgzip=rapidgzip:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - python
+    - setuptools >=46.4.0
+    - wheel
+    - cython >=0.29.24
+    - nasm
```

Comment:
nasm is only necessary during build not on the host. Also, does this section allow non-pip tools? Nasm should be installed from PyPI, only some weird unrelated project with the same name is there, it should be installed via yum. Maybe `{{ compiler('nasm') }}` would also work instead of the `yum_requirements` I used?

Reply:
My mistake! 🙏 
I checked other packages, and they include it just as `- nasm` in build requirements.